pymagicc/api.py

```diff
@@ -255,27 +254,46 @@ def check_config(self):
         ------
         ValueError
             If we are not certain that the config written by PYMAGICC will overwrite
-            all other config i.e. that there will be no unexpected behaviour.
+            all other config i.e. that there will be no unexpected behaviour. A
+            ValueError will also be raised if the user tries to user more than one
+            scenario file.
         """
         raise_cfg_error = False
+        raise_emisscen_error = False
```

Comment:
I think this pattern with multiple checks and raising ther error later can be confusing.
How about something like:
```python
multiple_cfg_files_found = "PYMAGICC is not the only tunin ..."
```

And then each time a check fails
```python
raise ValueError(multiple_cfg_files)
```python
Same for `raise_emisscen_error`,
multiple_scenarios_found = "ou have more than one `FILE_EMISSCEN_X` ..."
```